opencga-app/app/scripts/azure/arm/azuredeploy.json

```diff
@@ -165,6 +165,64 @@
             "metadata": {
                 "description": "Storage Account type"
             }
+        },
+        "mongoVmSize": {
+            "type": "string",
+            "metadata": {
+                "description": "Size of vm (e.g. Standard_D1_v2)"
+            },
+            "defaultValue": "Standard_D1_v2"
+        },
+        "mongoDnsLabelPrefix": {
+            "type": "string",
+            "metadata": {
+                "description": "Unique DNS Name for the Public IP used to access the Virtual Machine."
+            }
+        },
+        "mongoCertificateEmail": {
+            "type": "string",
+            "metadata": {
+                "description": "Email address used for mongoDB letsencrypt certificates"
+            }
+        },
+        "mongoClusterSize": {

```

Comment:
Have we got to have a replica set or can we have single VM option?

Reply:
Makes sense to add '1' to the list and keep the default option to have a 3 member replica set

Reply:
Added in https://github.com/opencb/opencga/pull/975/commits/df8d50b83256e033ac74c5dfc1539c166d3a5141